### ISSUE TITLE
Linkability between client tx and client IP in ETH 2.0

### ISSUE THREAD

**Issue by 29988122** [ethereum/go-ethereum#22592]
Paper:
https://nsg.ee.ethz.ch/fileadmin/user_upload/publications/perimeter.pdf
Talk:
https://www.youtube.com/watch?v=VCGyZADm_aQ

tl;dr:
This paper PERIMETER from Financial Cryptography and Data Security 2021 claimed that the behavior of devp2p in ETH 1.0 clients could cause the TXs from certain user to be associated with that exact user's ip address. 

Although we already have transport layer encryption [(rlpx)](https://github.com/ethereum/devp2p/blob/master/rlpx.md) from devp2p in ETH 1.0, traffic analysis can be applied as claimed by PERIMETER. 

As far as I know, the p2p interface in ETH 2.0 is more close to libp2p rather than devp2p. 
https://github.com/ethereum/eth2.0-specs/blob/dev/specs/phase0/p2p-interface.md

Does the noise protocol used by libp2p have the appropriate security features to deal with the traffic analysis issues found by PERIMETER? 
Or, is the issue in the paper actually a *non-issue*, in a sense that it doesn't matter in the broad picture?

All in all, just a FYI as I myself would also like the clients to be more secure in ETH 2.0. 
: P

**Comment by 29988122** [comment]
moved the issue to the correct repo
https://github.com/ethereum/eth2.0-specs/issues/2285
